src/utils/queries.js

```diff
@@ -0,0 +1,50 @@
+export const createPassengerTable = `
+CREATE TABLE IF NOT EXISTS passenger (
+  id SERIAL PRIMARY KEY,
+  "firstName" VARCHAR NOT NULL,
+  "lastName" VARCHAR NOT NULL,
+  "phoneNumber" VARCHAR NOT NULL,
+  password VARCHAR(250) NOT NULL,
+  email VARCHAR NOT NULL
+)
+  `;
+
+export const createDriverTable = `
+CREATE TABLE IF NOT EXISTS driver (
+  id SERIAL PRIMARY KEY,
+  "firstName" VARCHAR NOT NULL,
+  "lastName" VARCHAR NOT NULL,
```

Comment:
Your SQL naming is not consistent, you have underscore in some places and camel case for some  other columns. Please fix this.